### PR TITLE
3.28

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

File: src/Datagrid/Pager.php [hunk 1]
```diff
@@ -31,7 +31,7 @@ class Pager extends BasePager
     /**
      * NEXT_MAJOR: remove this property.
      *
-     * @deprecated since sonata-project/doctrine-orm-admin-bundle 3.x and will be removed in 4.0
+     * @deprecated since sonata-project/doctrine-orm-admin-bundle 3.28 and will be removed in 4.0
```
Comment (phansys) [inline src/Datagrid/Pager.php:34]: This deprecation is not listed in the changelog.

**Review by phansys — APPROVED** [review]

**Comment by VincentLanglet** [comment]
Test are failing randomly

This was already failing on 3.x after the last merge
https://github.com/sonata-project/SonataDoctrineORMAdminBundle/runs/1765962077

Can you take a look @franmomu ? :) 
Maybe related to https://github.com/symfony/panther/issues/169 ?

**Comment by franmomu** [comment]
👍  I'll try to take a look later today.

**Comment by VincentLanglet** [comment]
Seems like tests are green now. It fails randomly.

**Pull request merged**